src/main/java/net/minecraftforge/common/command/EntitySelectorManager.java

```diff
@@ -88,7 +88,7 @@ public static EntitySelector parseSelector(EntitySelectorParser parser) throws C
     }
 
     /**
-     * This method is called in {@link EntitySelectorParser#fillSelectorSuggestions}
+     * This method is called in {@code EntitySelectorParser#fillSelectorSuggestions(SuggestionsBuilder)}
```

Comment:
Inconsistency in code formatting, either fully qualified or simple.

Reply:
What do you mean?